crates/pallet-domains/src/runtime_registry.rs

```diff
@@ -42,15 +46,24 @@ pub enum Error {
 }
 
 /// Domain runtime specific information to create domain raw genesis.
-#[derive(TypeInfo, Debug, Encode, Decode, Clone, PartialEq, Eq, Copy)]
+#[derive(TypeInfo, Debug, Encode, Decode, Clone, PartialEq, Eq)]
 pub enum DomainRuntimeInfo {
-    EVM { chain_id: EVMChainId },
+    EVM {
+        /// The EVM chain id for this domain.
+        chain_id: EVMChainId,
+        /// Accounts initially allowed to create contracts on this domain.
+        /// The domain owner can update this list using a sudo call.
+        initial_contract_creation_allow_list: PermissionedActionAllowedBy<EthereumAccountId>,
+    },
```

Comment:
NOTE this changes the storage structure, we need storage migration to keep compatible with the existing EVM domain on Taurus

Reply:
This change to `DomainRegistry: DomainObject` storage also impacts `DomainObject.domain_config`, because the `DomainConfig` and `DomainRuntimeInfo` types are both changed.